.editorconfig

```diff
@@ -1,22 +1,19 @@
 root = true
 
 [*]
-charset=utf-8
+charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
+end_of_line = lf
 indent_style = space
 indent_size = 2
 
-[*.{py,sh,js,ps1}]
+[*.{py,sh,ps1,js,json,css}]
 indent_size = 4
-line_length = 120
+max_line_length = 120
 skip = external_libs
 known_first_party = lightgbm
 
-# Placeholder files
-[{*.gitkeep,__init__.py}]
-insert_final_newline = none
```

Comment:
Unfortunately, `biome` fails to read the config here. Also, [`insert_final_newline=unset|off`](https://github.com/biomejs/biome/pull/3907) doesn't help.

```
configuration ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  × Failed to parse the .editorconfig file.
    
    Caused by:
      Custom("expected 'true' or 'false'")
```
Anyway, [`none` is not allowed value in editorconfig](https://spec.editorconfig.org/), so these lines haven't actually worked.